### ISSUE TITLE
Questionable use of alerts in search view

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#95861]
I see that search uses `aria.alert` for example [here](https://github.com/microsoft/vscode/blob/07bd3fff5c576e539f3079c09a46d15d5af98338/src/vs/workbench/contrib/search/browser/searchView.ts#L822)

Why do we use alerts? Idealy the focus should just nicely flow through the tree and there should be no need to use `alert` since the tree elements should have proper aria labels set.

Assigning to May since I will have time to tackle then if you do not tackle this in the meantime.

**Comment by isidorn** [comment]
Also consider using `tree.ariaLabel` to dynamically update the `ariaLabel` of the whole tree based on the number of results there are.
We are doing the same in problems view.

**Comment by roblourens** [comment]
This is from https://github.com/microsoft/vscode/issues/91705. I don't know how else the user would know what happened, the effect of that command is just to move the editor to another place.

**Comment by isidorn** [comment]
@roblourens thanks for claryfing. Than I think it makes good sense.

Though still keeping this issue open since we can update the `ariaLabel` of the whole tree based on the term. For example in this [method](https://github.com/microsoft/vscode/blob/ac1a73d950087f94d6943150c42a8d7b4d0bb193/src/vs/workbench/contrib/search/browser/searchView.ts#L1555)
We could update `this.tree.ariaLabel` to be:
"Search: 'TERM' has NUMBER results in NUMBER_FILES files"

Let me know what you think

**Comment by roblourens** [comment]
Sure. I would maybe put the term last though since it could be long.

**Comment by isidorn** [comment]
Makes sense.

**Comment by isidorn** [comment]
Unassignign myself, let me know if you need help.
It should be as trivial as setting `this.tree.ariaLabel` somewhere around [here](https://github.com/microsoft/vscode/blob/dd3bf7169c68de4c62955621759ed788042afc1a/src/vs/workbench/contrib/search/browser/searchView.ts#L1468).

**Comment by JacksonKearl** [comment]
@roblourens I'll look into this

**Comment by jhomme** [comment]
Hi,
I would just remove the first part of the message that says "search found," because by context the user knows they are searching. The same idea would apply to searching within just one file.

> 
> 
> Unassignign myself, let me know if you need help.
> It should be as trivial as setting `this.tree.ariaLabel` somewhere around [here](https://github.com/microsoft/vscode/blob/dd3bf7169c68de4c62955621759ed788042afc1a/src/vs/workbench/contrib/search/browser/searchView.ts#L1468).
> I can also provide a PR if you prefer that.
